### PR TITLE
New pagination

### PR DESCRIPTION
**Pull request by i-like-robots**
The pagination currently sits alone in the long ignored stream 'footer'. This brings it into the stream and won't break when a user filters the page.

screen shot 2016-04-01 at 16 46 57

### CODE REVIEW

File: server/lib/format-query-string.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+'use strict';
+
+module.exports = (params) => {
+	const chunks = Object.keys(params)
+		.map(item => `${item}=${encodeURI(params[item])}`);
+
+	return `?${chunks.join('&')}`;
```
Comment (ironsidevsquincy) [inline server/lib/format-query-string.js]: could just add the `join` to the above and directly return

File: shared/components/pagination/index.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+const React = require('react');
+
+module.exports = (props) => {
```
Comment (ironsidevsquincy) [inline shared/components/pagination/index.js]: is this not actually a React component then?

Comment (i-like-robots) [inline shared/components/pagination/index.js]: Yep, it's just a 'stateless' one.

**Comment by ironsidevsquincy** [comment]
:+1:

**Comment by andygnewman** [comment]
LGTM 👍

**Pull request merged**